index.js

```diff
@@ -2,188 +2,202 @@
 
 const fs = require('fs')
 const path = require('path')
-const steed = require('steed')
 
-module.exports = function (fastify, opts, next) {
+module.exports = async function (fastify, opts) {
+  await loadAllPlugins(fastify, opts)
+}
```

Comment:
Unfortunately this still support Node.js 6. We can move to async/await it when we cut Fastify v3.

Reply:
I could refactor it to use Promises only.

Or we do not merge this until fastify v3 has been cut.

Reply:
@delvedor what do you think?

Reply:
Fastify v3 will be out by the end of the year (more or less).
I'm not using es modules so I don't see the urge to do this, but it may be useful to support them.
I'm fine with both options.

Reply:
Let's go with promises, I prefer this to be shipped asap so we can test it out.